src/app/shared/pricing-definitions/pricing-definition/pricing-definition.component.ts

```diff
@@ -326,7 +426,21 @@ export class PricingDefinitionComponent implements OnInit {
     }
   }
 
-  private consistencyCheck(pricingDefinition: PricingDefinition) {
+  private clearAndResetControl(control: AbstractControl) {
+    control.reset();
+    control.clearValidators();
+    control.updateValueAndValidity();
+  }
+
+  private checkPricingDefinition(pricingDefinition: PricingDefinition) {
+    if (this.timeFromEnabled.value) {
+      if (this.timeToValue.value === this.timeFromValue.value) {
```

Comment:
Should be displayed in the UI, check how password is handled.

Reply:
it's displayed when we save already

Reply:
The error message should be displayed in the UI and not when you click on save.
But I think you did it already in your latest changes, right?

Reply:
This was the first feedbacks from Gerald.
Having it per hour and receiving the Meter Values and pricing them per min would almost always result in a floating point.
Furthermore, the meter value interval can be changed (one every 5 mins).
Maybe a drop-down with 'mins' and 'hours' would be the solution :-).

Reply:
> > 
> 
> ok but in a the next PR lol

Ca fait la deuxieme fois que t'essaies de me la faire a l'envers lol
If it's just a front-end change do it right now ;-P

Reply:
> I need customer feedbacks ... the feeling from Gerald is not sufficient. Especially when I see the mess that we have in our own Gireve Tariff definitions.

Even Kinza expected mins and got lost and this was also a feeling I had.

> I think this is not so simple. Where do we store the fact that the user entered it per minutes and not in hours?

This was my initial question, if it requires a change in the backend we can postpone it but not forever ;-).